src/platform/elements/select/Select.ts

```diff
@@ -31,7 +31,7 @@ const SELECT_VALUE_ACCESSOR = {
   selector: 'novo-select',
   providers: [SELECT_VALUE_ACCESSOR],
   template: `
-    <div (click)="openPanel()" tabIndex="0" type="button" [class.empty]="empty">{{selected.label}}<i class="bhi-collapse"></i></div>
+    <div (focus)="openPanel()" tabIndex="0" type="button" [class.empty]="empty">{{selected.label}}<i class="bhi-collapse"></i></div>
```

Comment:
So, if the element is already focused then the user cannot open it again?

Open the select, select an element (select still has focus) and you cannot select it again.

Reply:
Good catch; I changed it to now if the user clicks on it again, presses up, or presses down while the select is focused, the panel is opened again. I'll get with Jessica today to talk about the changes.